util/embedders.py

```diff
@@ -37,7 +37,7 @@ def get_embedder(
             elif model == "bag-of-words":
                 embedder = BagOfWordsSentenceEmbedder(batch_size=batch_size)
             elif model == "tf-idf":
-                embedder = TfidfSentenceEmbedder(batch_size=batch_size)
+                embedder = TfidfSentenceEmbedder(batch_size=batch_size, min_df=0)
```

Comment:
We could also return the embedder here directly similar to the bag-of-chairs embedder to follow the same pattern. Or the otherway around returning the bag-of-chair in the end.

Reply:
changed to return